packages/svelte/src/reactivity/set.js

```diff
@@ -86,14 +87,30 @@ export class ReactiveSet extends Set {
 		return super.has(value);
 	}
 
+	#read_all() {
+		var sources = this.#sources;
+		var values = super.values();
```

Comment:
We shouldn't create this iterator eagerly, it can go inside the if block

Reply:
Gosh I need to take a vacation. Will fix later.

Reply:
at my laptop now so i took care of it

Reply:
Thanks 😁